vm.go

```diff
@@ -536,65 +563,158 @@ func (v *VM) run() {
 				v.sp++
 			}
 		case parser.OpCall:
-			numArgs := int(v.curInsts[v.ip+1])
-			spread := int(v.curInsts[v.ip+2])
-			v.ip += 2
+			var (
+				numArgs           = int(v.curInsts[v.ip+1])
+				hasVarArgs        = int(v.curInsts[v.ip+2])
+				numKws            = int(v.curInsts[v.ip+3])
+				hasVarKw          = int(v.curInsts[v.ip+4])
+				kwargs, varKwargs map[string]Object
```

Comment:
I believe that the declarations of the variables influence the performance, but are needed. I do not know how to improve this.